pkg/interfaces/contracts/standalone-utils/IProtocolIdRegistry.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+
+/**
+ * @dev Registry of protocol IDs for external integrations with Balancer. The IDs chosen are arbitrary and do not affect
+ * behavior of any Balancer contracts. They are used only to tag specific contracts (usually pools) at the data layer.
+ */
+interface IProtocolIdRegistry {
+    // Emitted when a new protocol ID is registered.
+    event ProtocolIdRegistered(uint256 indexed protocolId, string name);
+
+    /**
+     * @dev Registers an ID (and name) to differentiate among protocols. Protocol IDs cannot be deregistered.
+     */
+    function registerProtocolId(uint256 protocolId, string memory name) external;
+
+    /**
+     * @dev Returns true if `protocolId` has been registered and can be queried.
+     */
+    function isValidProtocolId(uint256 protocolId) external view returns (bool);
+
+    /**
+     * @dev Returns the name associated with a given `protocolId`.
+     */
+    function getProtocolName(uint256 protocolId) external view returns (string memory);
+}
```

Comment:
I'd move this to the `interfaces` package (update the changelog if you do!)

Reply:
@nventuro This is in the ```interfaces``` package already. Unless you are referring to a different ```interfaces``` package?

Reply:
I'm dumb and didn't notice :sob: